app/src/main/java/com/bachelor_group54/funnregistrering/MainActivity.java

```diff
@@ -215,4 +213,8 @@ public void toLoginPageBtn(View view){
         fragmentTransaction.addToBackStack(""); //Legger Fragmentet på stack så back knappen fungerer
         fragmentTransaction.commit();
     }
+    public void showPreferencesBtn(View veiw){//settings/preference page button. Creates an intent using the SetPreferenceActivity class and starts an activity.
```

Comment:
Endrer veiw til view
```suggestion
    public void showPreferencesBtn(View view){ //settings/preference page button. Creates an intent using the SetPreferenceActivity class and starts an activity.
```